website/docs/reference/warehouse-profiles/postgres-profile.md

```diff
@@ -55,11 +56,15 @@ The `role` config controls the Postgres role that dbt assumes when opening new c
 
 The `sslmode` config controls how dbt connectes to Postgres databases using SSL. See [the Postgres docs](https://www.postgresql.org/docs/9.1/libpq-ssl.html) on `sslmode` for usage information. When unset, dbt will connect to databases using the Postgres default, `prefer`, as the `sslmode`.
 
-### Postgres notes
-### Performance
-While Postgres works reasonably well for datasets smaller than about 10mm rows, database tuning is sometimes required. Make sure to create indexes for columns that are commonly used in joins or where clauses.
-
 ### `keepalives_idle`
 If the database closes its connection while dbt is waiting for data, you may see the error `SSL SYSCALL error: EOF detected`. Lowering the [`keepalives_idle` value](https://www.postgresql.org/docs/9.3/libpq-connect.html) may prevent this, because the server will send a ping to keep the connection active more frequently. 
 
-[dbt's default setting](https://github.com/dbt-labs/dbt-core/blob/main/plugins/postgres/dbt/adapters/postgres/connections.py#L28) is 0 (the server's default value), but can be configured lower (perhaps 120 or 60 seconds), at the cost of a chattier network connection.
\ No newline at end of file
+[dbt's default setting](https://github.com/dbt-labs/dbt-core/blob/main/plugins/postgres/dbt/adapters/postgres/connections.py#L28) is 0 (the server's default value), but can be configured lower (perhaps 120 or 60 seconds), at the cost of a chattier network connection.
+
+<VersionBlock firstVersion="1.2">
+
+#### retries
+
+If `dbt-postgres` encounters an operational error or timeout when opening a new connection, it will retry up to the number of times configured by `retries`. The default value is 1 retry. If set to 2+ retries, dbt will wait 1 second before retrying. If set to 0, dbt will not retry at all.
```

Comment:
We could add a caveat that due to a psycopg2 limitation, some non-retryable errors like invalid credentials will be retried as well. But it's not strictly necessary to include.

Reply:
> If set to 2+ retries, dbt will wait 1 second before retrying.

only true if we don't merge https://github.com/dbt-labs/dbt-core/issues/5502 which adds exponential backoff to both postgres and redshift.

Reply:
> only true if we don't merge https://github.com/dbt-labs/dbt-core/issues/5502 which adds exponential backoff to both postgres and redshift.

Yup! I assumed that we'd be making that change for v1.3 (at this point, since v1.2.0-rc2 is already released), at which point we should swing back to update these docs.